src/services/member.ts

```diff
@@ -80,3 +80,14 @@ export async function updateMemberProfile(
 
   return deserializeMember(res.data);
 }
+
+// TODO: please god give me an endpoint for this
+export async function isCurrentMemberVerified(): Promise<boolean> {
```

Comment:
You can just poll the member and if it's part of the group it's fine no?

Reply:
yeah that's what this is doing rn, just feels silly to poll large object for 1 bit of data